keyboards/massdrop/alt/keymaps/pregame/readme.md

```diff
@@ -0,0 +1,35 @@
+# Massdrop Alt Pregame
+
+This keymap is here to show some love to the MD Alt. This is a stripped down version of the Drop Ctrl Endgame; hence the name Pregame.
+
+## IRL
+![In Real Life](https://i.imgur.com/Xp6Mb6P.jpg)
+
+
+
+### Layers
+
+## Typing Layer
+
+![Typing Layer](https://i.imgur.com/F7iU53K.png)
+
+## Function Layer
+
+![SHORTCUT_LAYER](https://i.imgur.com/Gub1xyC.png)
+
+## Features
+
+These are the features currently supported by this keymap:
+
+# RGB Can be customized by "NAME OF COLOR" in the Keymap.c file. 
```

Comment:
One of your rebase commits changed the formatting of the headers down here:

![image](https://user-images.githubusercontent.com/18669334/99146409-be6a8300-262c-11eb-98ad-cd45434bdfa3.png)

```suggestion
### RGB Can be customized by "NAME OF COLOR" in the Keymap.c file. 
```